scripts/bisect.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+echo $(pwd)
+
+
+while getopts "e:f:hl:t:" opt; do
+  case "$opt" in 
+    e)
+      exp_file="$OPTARG"
+      echo "exp file is $exp_file"
+      ;;
+    l)
+      libc="$OPTARG"
```

Comment:
Required params like these should be forced to be set (exit otherwise).
Otherwise a user could run `make report-linux` and only after that run into an error, which isn't the best user experience when kicking off a bisection (probably run overnight).

Something like this below the getopts might work:
```
if [ -z "$exp_file" ] || [ -z "$libc" ] || [ -z "$test_case" ] || [ -z "$target_arch" ]; then
        echo 'Missing required parameter. Run with -h to see all parameters.' >&2
        exit 1
fi
```
[reference](https://unix.stackexchange.com/questions/621004/bash-getopts-mandatory-arguments)